tests/tests_unit/rules/test_importers/test_inference_importer.py

```diff
@@ -8,13 +8,20 @@
 
 
 def test_rdf_inference():
-    rules, _ = InferenceImporter.from_rdf_file(nordic44_knowledge_graph).to_rules(errors="continue")
+    store = NeatGraphStore.from_oxi_store()
+    extractor = RdfFileExtractor(nordic44_knowledge_graph, base_uri="http://nordic44.com/")
+    store.write(extractor)
+
+    rules = InferenceImporter.from_graph_store(store).to_rules(errors="raise")
 
     assert len(rules.properties) == 312
     assert len(rules.classes) == 59
 
     # checking multi-value type
-    assert set(rules.properties.data[19].value_type.types) == set(
+    multi_value_property = "OperatingShare.PowerSystemResource"
+    prop = next((prop for prop in rules.properties if prop.property_ == multi_value_property), None)
+    assert prop is not None, f"Failed to infer expected multi-value property {multi_value_property}"
+    assert set(prop.value_type.types) == set(
```

Comment:
oxy seems to be not be deterministic, so need a most robust check here

Reply:
The order of processing ?

Reply:
Running this tests a few times the "OperatingShare.PowerSystemResource" property is not always on index 19, it changes from run to run.

Reply:
that is expected, I think with in-memory rdflib store they do indexing of triples or something similar to maintain the order